app/js/mongodb.js

```diff
@@ -2,18 +2,21 @@ const Settings = require('settings-sharelatex')
 const { MongoClient, ObjectId } = require('mongodb')
 
 const clientPromise = MongoClient.connect(Settings.mongo.url)
-const dbPromise = clientPromise.then((client) => client.db())
-
-async function getCollection(name) {
-  return (await dbPromise).collection(name)
-}
 
 async function waitForDb() {
   await clientPromise
 }
 
+const db = {}
+waitForDb().then(async function () {
+  const internalDb = (await clientPromise).db()
+
+  db.docs = internalDb.collection('docs')
+  db.docOps = internalDb.collection('docOps')
+})
+
 module.exports = {
+  db,
   ObjectId,
-  getCollection,
```

Comment:
@gh2k I just noticed that this will break your script, which is using the `getCollection` helper to receive the `projects` collection -- which is not exported on `db` in this PR. To not complicate things, WDYT about creating a new `MongoClient` in the script?

Reply:
Yes, that makes sense. I'm not running the script while my computer is broken so it's fine to update.